docs/sql-ref-syntax-ddl-create-view.md

```diff
@@ -19,4 +19,69 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+Views are based on result-set of complex base `SQL` queries that are hard to
+maintain or understand. `CREATE VIEW` constructs a virtual table that has no
+physical data therefore other operations like `ALTER VIEW` and `DROP VIEW` 
+only change metadata. 
+
+### Syntax
+{% highlight sql %}
+CREATE [OR REPLACE] [[GLOBAL] TEMPORARY] VIEW [IF NOT EXISTS] [db_name.]view_name
+    [(column_name [COMMENT column_comment], ...) ]
+    create_view_clauses
+    AS SELECT ...;
+    
+    create_view_clauses (order insensitive):
```

Comment:
Can we treat `create_view_clauses` a parameter ? 
Also the query portion can be made a parameter as well ..